docs/root/configuration/runtime.rst

```diff
@@ -145,6 +151,29 @@ built into the code, except for any values added via `/runtime_modify`.
   Changes are effectively immediately. It is **critical** that the admin interface is :ref:`properly
   secured <operations_admin_interface_security>`.
 
+At most one admin layer may be specified. If a non-empty :ref:`layered runtime
+<envoy_api_msg_config.bootstrap.v2.LayeredRuntime>` bootstrap configuration is specified with an
+absent admin layer, any mutating admin console actions will elicit a 503 response.
+
+.. _config_runtime_atomicity:
+
+Atomicity
+---------
+
+The runtime will reload and a new snapshot will be generated in a variety of situations, i.e.:
+
+* When a file move operation is detected under the symlink root or the symlink root changes.
+* When an admin console override is added or modified.
+
+All runtime layers are evaluated during a snapshot. Layers with errors are ignored and excluded from
```

Comment:
Is this consistent with the rest of envoy disk based config?  If I'm understanding it correctly I think this is pretty dangerous and we should revisit.  Happy to hash out in person.

Reply:
This is the Envoy disk based config :) This is basically the behavior we had before, see https://github.com/envoyproxy/envoy/blob/master/source/common/runtime/runtime_impl.cc#L514

Reply:
I left a TODO in `runtime_impl.cc` for this, since this isn't the right PR to change the existing behavior.